packages/shared/components/atoms/tiles/ActivityTile.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="typescript">
+    import {
+        InclusionState,
+        selectedAccountAssets,
+        getAssetFromPersistedAssets,
+        IPersistedAsset,
+        Activity,
+        ActivityType,
+        NotVerifiedStatus,
+    } from '@core/wallet'
+    import { openPopup } from '@lib/popup'
+    import {
+        ClickableTile,
+        TransactionActivityTileContent,
+        FoundryActivityTileContent,
+        AliasActivityTileContent,
+    } from 'shared/components'
+
+    export let activity: Activity
+
+    let asset: IPersistedAsset
+    $: $selectedAccountAssets, (asset = getAssetFromPersistedAssets(activity.data.assetId))
+
+    function handleTransactionClick(): void {
+        if (asset?.verification?.status === NotVerifiedStatus.New) {
+            openPopup({
+                type: 'tokenInformation',
+                overflow: true,
+                props: {
+                    activityId: activity.id,
+                    asset,
+                },
+            })
+        } else {
+            openPopup({
+                type: 'activityDetails',
+                props: { activityId: activity.id },
+            })
+        }
+    }
+</script>
+
+<ClickableTile
```

Comment:
I think that the bottom section is for async transactions and can  be shared across technically any output not just transactions. So that and the claim handling can be in this parent component.